host/onebox.go

```diff
@@ -338,10 +350,23 @@ func (c *cadenceImpl) startFrontend(hosts map[string][]string, startWG *sync.Wai
 	c.frontEndService = service.New(params)
 	c.adminHandler = frontend.NewAdminHandler(
 		c.frontEndService, c.historyConfig.NumHistoryShards, c.metadataMgr, c.historyMgr, c.historyV2Mgr)
-	frontendConfig := frontend.NewConfig(dynamicconfig.NewCollection(params.DynamicConfig, c.logger), false)
+	dc := dynamicconfig.NewCollection(params.DynamicConfig, c.logger)
+	frontendConfig := frontend.NewConfig(dc, c.esConfig.Enable)
+	visibilityMgr := c.visibilityMgr
+	if c.esConfig.Enable && c.enableReadVisibilityFromES {
```

Comment:
I think we should remove `enableReadVisibilityFromES` config from test and only use esEnable as single switch for both write and read with ES. Because we probably always want both rw

Reply:
Removed.